google_drive/tests/unit/test_docs.py

```diff
@@ -0,0 +1,240 @@
+""" Tests for google drive components """
```

Comment:
Me too! (a little)
Google Drive is the overall concept; Google Docs are the document types that you can store on your Google Drive. Calendars are the other things you can install on Google Drive.
(This is actually true as of early 2014, when google changed the branding / concept of what Google Drive was, it used to just be a synched-drive)